docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+- `certificate-ttl` is the option that will determine the time until expiration for any `tbot` issued certificates, including host certificates. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
+- `auth-server` is the address of your Teleport Auth Server, for example
+  `auth.example.com:3025`. If your Machine ID host is in a different network
+  than your Auth Server, use the public web address of your Proxy Service
+  instead (e.g., `auth.example.com:443`).
+
+
+</ScopedBlock>
+
+Using a text editor of your choice, write the output generated by the `tbot configure` command to a configuration YAML file, in this example we'll be using `tbot.yaml`.
+
+Once complete, the Machine ID configuration will still require a destination directory to be manually referenced within the configuration file. The destination directory is where Machine ID will write certificates
+which can be used by application and tools to start Machine ID. In the configuration yaml `tbot.yaml`, this configuration will additionally be used to designate the **principal**,
+in this case the Domain Name that we'll be applying to the tbot configuration to access a Node.
+The new configuration YAML will require a `destination` field and `ssh_host_cert` subfield that resembles the following, replacing `nodename.my.domain.com` with your own required domain name or IP addresses:
+
+```
+destinations:
+  - directory:
+      path: /opt/machine-id
+    configs:
+      - ssh_host_cert:
+          principals: [nodename.my.domain.com]
+```
+
+Once completed, your `tbot.yaml` configuration should resemble the following:
+
+```
+onboarding:
+  token: "1234abcd5678efgh9"
+  ca_path: ""
+  ca_pins:
+  - sha256:1234abcd5678efgh910ijklmnop
+  join_method: token
+storage:
+  directory:
+    path: /var/lib/teleport/bot
+    symlinks: secure
+    acls: try
+destinations:
+  - directory:
+      path: /opt/machine-id
+    configs:
+      - ssh_host_cert:
+          principals: [nodename.my.domain.com]
+debug: false
+auth_server: example.teleport.sh:443
+certificate_ttl: 1h0m0s
+renewal_interval: 20m0s
+oneshot: false
+```
+
+Note the `renewal_interval` field, which will determine the cadence in which new certificates will be generated. Machine ID uses a default of 20 minutes for the `renewal_interval`, however this field may be adjusted as needed.
+
+## Step 4/4 Start Machine ID and create a host certificate
+
+Using your configuration file, The `tbot start` command will start running Machine ID in a loop,
+writing renewable certificates to `/var/lib/teleport/bot` according to your configuration, and the short-lived certificates including the Host Certificate your bot will use to `/opt/machine-id`.
+
+In a production environment you will want to run Machine ID in the background using a service manager like systemd. However, in this guide you will run it in the foreground to better understand how it works.
+
+```code
+$ tbot start -c tbot.yaml
```

Comment:
Should we add a debug/verbose flag here?

Reply:
The command outputs information by default, I think that a verbose/debug flag may cause confusion unless we're highlighting some specific output we're otherwise not seeing

Reply:
I agree with @rsyracuse , it's probably best to not include `-d` here for standard usage as the additional logging won't be particularly useful.